phoenix-hbase-compat-2.0/src/main/java/org/apache/phoenix/compat/hbase/CompatUtil.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
```

Comment:
Formatting on this license header, please.

Reply:
Done